botocore/docs/utils.py

```diff
@@ -177,3 +178,18 @@ def append_documentation(self, event_name, section, **kwargs):
             description_section = section.get_section(
                 'param-documentation')
             description_section.writeln(self._doc_string)
+
+
+_CONTROLS = {
+    '\n': '\\n',
```

Comment:
According to json.org the following are control characters:
```
\b
\f
\n
\r
\t
```

I'll add all of these just in case, but as far as I can tell the three there currently are the only ones that will break sphinx rendering.